src/litserve/server.py

```diff
@@ -414,7 +414,7 @@ async def health(request: Request) -> Response:
 
         @self.app.get(self.info_path, dependencies=[Depends(self.setup_auth())])
         async def info(request: Request) -> Response:
-            return JSONResponse(
+            return ORJSONResponse(
```

Comment:
this is not a performance critical section. I would suggest applying this to the predict method.

Reply:
Got it! In the latest commit I'm hard-coding the `ORJSONResponse` class while registering the `/predict` endpoint, but it's only temporary for the sake of the benchmarks. In the target scenario this would be user's choice